packages/iris-grid/src/formatters/TableColumnFormatter.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint class-methods-use-this: "off" */
+/* eslint no-unused-vars: "off" */
+/**
+ * Default column data formatter. Just interpolates the value as a string and returns.
+ * Extend this class and register with TableUtils to make use of it.
+ */
+
+export type TableColumnFormatType =
+  | 'type-global'
+  | 'type-context-preset'
+  | 'type-context-custom';
+
+export type TableColumnFormat = {
+  label: string;
+  formatString: string;
+  type: TableColumnFormatType;
+};
+
+export class TableColumnFormatter<T = unknown> {
+  static TYPE_GLOBAL: TableColumnFormatType = 'type-global';
+
+  static TYPE_CONTEXT_PRESET: TableColumnFormatType = 'type-context-preset';
+
+  static TYPE_CONTEXT_CUSTOM: TableColumnFormatType = 'type-context-custom';
+
+  /**
+   * Validates format object
+   * @param format Format object
+   * @returns true for valid object
+   */
+  static isValid(format: TableColumnFormat): boolean {
+    return true;
+  }
```

Comment:
Can you add a comment that says why this just returns true? Is it something that is a TODO, is it something that should be added by the consumer of this class? Something we were initially checking client-side but switched to server-side and it's just for maintaining existing code?

Reply:
Nevermind I saw the comment at the top of the file saying it's the default formatter

Should we maybe use `abstract` classes and methods in this case? At least the class should be `abstract` I think if it's supposed to be extended and not directly used. Not sure if we want defaults for the methods or mark some `abstract` so the implementation has to define it

Reply:
Yea it _should_ be an abstract class, or possibly even just an interface. I added a comment on #21 to consider that change. I didn't want to change any of the existing functionality as I was just changing the one class so I don't get errors in Linker